### DISCUSSION TITLE
Animal territory model animation

### DISCUSSION THREAD

**Discussion by taoyun** [usnistgov/fipy#912] [Show and tell]
Currently leading a paper promoting FiPy to modelers in ecology and epidemiology.

One of the paper's demo is this animation showing the territory formations of four wolf packs. Contour lines indicate their space use; color represents collective scent mark density.

https://user-images.githubusercontent.com/1597175/226963094-d0202d83-d146-4a96-8b65-526e0e5976fe.mp4

**Comment by guyer** [comment]
Thanks for sharing!
